pyomo/contrib/viewer/ui.py

```diff
@@ -46,6 +46,10 @@ class _MainWindowUI(object):
     class _MainWindow(object):
         pass
 
+if not qt_available:
+    _log.error("Qt is not available. Cannot create UI classes.")
+    sys.exit(1)
```

Comment:
I missed this the first time around, but I think that this "`if`" needs to be re-thought.  The current implementation will forceably kill the Python interpreter if `not qt_available`.  That will wreak havoc in our slim builds and for any users that do not have Qt.

Reply:
Yeah.  You're right I'll raise an import error.

Reply:
Replaced the with raise import error and tested without PyQt.  The result is a pretty easy to understand exception.  Everything seems good.